src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function Navbar() {
+  return (
+    <nav className="navbar">
+      <div className="logo">
+        <h2>valoTube</h2>
+      </div>
+      <div className="right-nav">
+        <ul>
+          <li>Github</li>
+          <li>
+            <span>Guest</span>
+          </li>
```

Comment:
What is the significance of enclosing guests in `span` tag and Github only in `li`? You could avoid these to maintain consistency unless necessary

Reply:
I tried giving an avatar searching for inspiration on dribble.
But abandoned the idea after sometime.